NetCoreDemos/Views/Splitter/Overview.cshtml

```diff
@@ -66,25 +71,26 @@
         return `${resizableText} and ${collapsibleText}`;
     }
 
-    function paneContentTemplate(data, paneName) {
+    function paneContentTemplate(data, paneName, element) {
         const dimensionOptions = new Set(['size', 'minSize', 'maxSize']);
         const filteredData = Object.fromEntries(
             Object.entries(data)
                 .filter(([key]) => dimensionOptions.has(key))
         );
+        element.eq(0).attr('tabIndex', '0');
 
         return `
-            <div class='pane-content' tabindex="0">
-                <div class='pane-title'>
-                    ${paneName}
-                </div>
-                <div class='pane-state'>
-                    ${getStateText(data)}
+                <div class='pane-content'>
```

Comment:
There are no any styles for border. Should they be presented in this PR?

Reply:
[f3b3d38](https://github.com/DevExpress/devextreme-demos/pull/3206/commits/f3b3d380de5365140a538df371a86eccaa4da34c)